packages/localize/README.md

```diff
@@ -609,6 +625,54 @@ export default locales.map(({locale, localeTransformer}) => ({
 }));
 ```
 
+### JavaScript example
```

Comment:
Not for this PR, but I'd consider breaking the readme up into multiple files, or paring it down and putting most of this on lit.dev

Reply:
I agree! I plan to move all of this content to https://lit.dev/docs/libraries/localization/ pretty soon